src/script/env-script.tsx

```diff
@@ -35,9 +36,15 @@ export const EnvScript: FC<EnvScriptProps> = ({ env, nonce }) => {
     nonceString = nonce;
   }
 
+  const strategy =
+    process.env?.NEXT_RUNTIME_ENV === 'TEST' ||
+    process.env?.NEXT_PUBLIC_RUNTIME_ENV === 'TEST'
+      ? 'afterInteractive'
+      : 'beforeInteractive';
```

Comment:
Why don't we want `beforeInteractive` for tests?

Reply:
The `Script` tag from Next doesn't seem to work in the tests (this is most likely a skill issue of me). I've now mocked the return of the `<script>` tag that we had before.